### PR TITLE
fix README

### PR DESCRIPTION
**Pull request by rnmitchell**
Minor typos in README

### CODE REVIEW

**Comment by rnmitchell** [comment]
Not sure how I did it, but it seems to be fixed now even though it's not showing up as a difference? (I think likely because I made the change on the master branch before creating a new branch).

There was another sentence I removed as well. But I think this is ready for review and merge @standage

**Comment by rnmitchell** [comment]
wait hold on a sec. Not ready yet!

File: lusSTR/wrappers/snps_convert.py [hunk 1]
```diff
@@ -61,7 +61,6 @@ def create_sample_df(indiv_df, output_type, all_col):
         .unstack(0)
         .reset_index()
     )
-    compiled_table.to_csv("test.csv", index=False)
```
Comment (rnmitchell) [inline lusSTR/wrappers/snps_convert.py:64]: Whoopsies, left in some debugging code 😄

**Comment by rnmitchell** [comment]
Since I somehow completely skipped any information about the SNP workflow... I rewrote a lot of the README to be more informative about the SNP workflow.

This is now ready @standage (I hope!).

**Review by standage — APPROVED** [review]

**Pull request merged**